### PR TITLE
Continuous Integration

### PR DESCRIPTION
**Pull request by thedrow**
This is the MVP for the CI workflow.
We'll re-iterate on it later.

### CODE REVIEW

**Comment by thedrow** [comment]
FYI, the tests are expected to fail for the time being.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip
+        restore-keys: |
+          ${{ runner.os }}-pip
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel nox 'poetry==1.2.0.a2' poetry-export-plugin
+    - name: Cache poetry
+      uses: actions/cache@v1
```
Comment (auvipy) [inline .github/workflows/tests.yml:33]:
```suggestion
      uses: actions/cache@v2.1.7
```

Comment (thedrow) [reply .github/workflows/tests.yml:33]: Why do we need a specific version and not just v2?

Comment (auvipy) [reply .github/workflows/tests.yml:33]: not exactly specific, just suggesting latest stable release

**Review by naomielst — APPROVED** [review]

**Pull request merged**